bobtimus/src/loan.rs

```diff
@@ -531,17 +557,18 @@ mod tests {
                 days: 30,
                 interest_mod: Decimal::ZERO,
             }],
-            collateralizations: vec![],
+            // not relevant for test but to show as reference - that's what the borrower can chose
+            initial_ltvs: vec![Ltv { ltv: dec!(0.70) }],
 
             // irrelevant for this test
```

Comment:
I suppose this comment should be removed given that you fill in values?

Reply:
It's still irrelevant but  I want to have the values there nevertheless :)